perllib/FixMyStreet/Roles/CobrandBulkyWaste.pm

```diff
@@ -459,4 +463,14 @@ sub _bulky_send_reminder_email {
 
 sub bulky_send_before_payment { 0 }
 
+sub bulky_per_item_pricing_by_property_type { 0 }
+
+sub bulky_per_item_price_key {
+    my $self = shift;
+    return 'price' if !$self->bulky_per_item_pricing_by_property_type;
```

Comment:
You could remove the need for `bulky_per_item_pricing_by_property_type` by having this check check `bulky_per_item_pricing_property_types` directly and seeing if it has any length or not.